pysat/instruments/jro_isr.py

```diff
@@ -57,7 +57,7 @@
 jro_fname1 = 'jro{year:4d}{month:02d}{day:02d}'
 jro_fname2 = '.{version:03d}.hdf5'
 supported_tags = {ss:{'drifts':jro_fname1 + "drifts" + jro_fname2,
-                      'drifts_ave':jro_fname1 + "drifts_ave" + jro_fname2,
+                      'drifts_ave':jro_fname1 + "drifts_avg" + jro_fname2,
```

Comment:
I'm curious why the testing didn't catch this typo.

Reply:
That's a good question.

Reply:
Previously, download failures weren't strictly flagged as an issue. The unit test printed a note but moved on. In pull #143 the download check now has to pass or specifically be exempted.
For this particular issue, the download should have been ok but the typo in list files would report that there are no files, which is interpreted as a download failure (now).